src/partinverse.jl

```diff
@@ -0,0 +1,38 @@
+# default methods for when an inverse exists
+export r_inv, l_inv, retraction, coretraction
+
+"""
+    r_inv(function)
+    retraction(function)
```

Comment:
Do we really need an alias? I prefer a simple API and would suggest defining only `right_inverse`.
```suggestion
```

Reply:
I agree, `right_inverse` and `left_inverse` would be best, I think.